src/utils/validate-redirect-url.ts

```
@@ -148,7 +148,8 @@ export function validateRedirectUrl(
 
 export function validateUrl(allowedUrls: string[], redirectUri?: string) {
   if (!redirectUri) {
-    return;
+    // I'm not sure about this one... I'm matching existing tests
+    return false;
```

I feel like this should throw an exception if it's not provided...

---

It depends if it's required or not. As the param is optional I would almost say that it should return true?